mewarpx/mewarpx/mwxrun.py

```diff
@@ -167,6 +167,21 @@ def _set_grid_params(self):
         else:
             raise ValueError("Unrecognized type of pywarpx.picmi Grid.")
 
+    def get_domain_area(self):
+        """Return float of simulation domain area in X & Y directions or R depending
+        on geometry."""
```

Comment:
This looks like it's calculating a cross-sectional area? Which is equivalent to the surface area over which current is emitted/absorbed? That seems to be what the callers use it for. Please describe it here.